packages/matter.js/src/cluster/server/ClusterServer.ts

```diff
@@ -96,7 +96,15 @@ export type ClusterServerObj<A extends Attributes, C extends Commands> =
         _commands: CommandServers<C>;
 
         /**
-         * Set Storage context sed by this cluster
+         * Assign this cluster to a specific endpoint
+         * @private
+         *
+         * @param endpoint Endpoint to assign to
+         */
+        _assignToEndpoint: (endpoint: Endpoint) => void;
```

Comment:
See the JSDoc aove ... "object" do not have any protected or provate methods ... anything is public. So I decided this "naming convention" together with the jsdoc to state this. Thats a pitfall of not being able to use real objects because of the too dynamic built interface